cql3/restrictions/restriction.hh

```diff
@@ -74,6 +130,7 @@ protected:
     enum_set<op_enum> _ops;
     target _target = target::SINGLE_COLUMN;
 public:
+    ::shared_ptr<wip::expression> wip_equivalent;
```

Comment:
Let's rename this to `expression`. Rather than coming up with a logical explanation about why, I'll just say I have an a strong aversion to merging partially working code and if it isn't renamed "wip" it will be easier for me to overcome it.

Reply:
Done (though I liked the descriptive original name).